news/2017-11-11-osu-idol-2017-finals-community-voting.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "osu!idol 2017 Finals - Community Voting!"
+date: 2017-11-11 00:00:00 +0000
+---
+
+It's finals time. Been a while since [last year's voting](https://osu.ppy.sh/home/news/2016-10-25-osuidol-2016-finals-community-voting) where the community chose [Slyleaf](https://osu.ppy.sh/users/3322032) to be the osu!idol of 2016 with [Daikyi](https://osu.ppy.sh/users/811832) and [MissTitannia](https://osu.ppy.sh/users/4490361) taking 2nd and 3rd place respectively. Now it is time for this year's community voting! We have three new finalists for you to hear - [Thievley](https://osu.ppy.sh/users/4717672), [Will Stetson](https://osu.ppy.sh/users/4909088) and [Renril](https://osu.ppy.sh/users/4955122)! Give 'em a listen and vote for your favourite!
```

Comment:
This is good, but a lot of info for an opener. Split the stuff about last year's osu!idol into it's own heading along with a brief explanation of what the contest is about, etc, for people who may not know what it is.